pkg/apis/velero/v1/backup.go

```diff
@@ -100,10 +100,18 @@ type BackupSpec struct {
 
 	// DefaultVolumesToRestic specifies whether restic should be used to take a
 	// backup of all pod volumes by default.
+	//
+	// Deprecated: this field is no longer used and will be removed entirely in v2.0.
```

Comment:
This is trivial but could you please rephrase to removed in future release, b/c we are not sure if it's v2.0 or other version.

Additionally, IMO we should also suggest user use `DefaultVolumesToFsBackup` instead.

Reply:
Done